src/buffer/channel.rs

```diff
@@ -50,10 +50,20 @@ pub fn view<'a>(
                             user.clone(),
                         )
                         .map(scroll_view::Message::UserContext);
+                        let row_style = match clients.connection(&state.server) {
+                            Some(conn)
+                                if user.nickname() != conn.nickname()
+                                    && message.text.contains(&conn.nickname().to_string()) =>
```

Comment:
Just to confirm, is there an action here for me specifically or is this just a to-do later on?

Reply:
I can resolve this since it's a bit more of a complex optimization with borrowing / lifetimes. I'll leave this PR open and tackle this once I'm home later today or tomorrow.

This is important because we're right in the most critical hot path of the app right here. These scrollables have a huge perf impact.

Reply:
No problem, thanks for looking at it!